app/src/marketplace/modules/dataUnion/services.js

```diff
@@ -287,7 +288,6 @@ export const deployDataUnion2 = (productId: ProductId, adminFee: string): SmartC
             return client.deployDataUnion({
                 dataUnionName: productId,
                 adminFee: +adminFee,
-                joinPartAgents: getStreamrEngineAddresses(),
```

Comment:
I don't think `joinPartAgents` can be removed 🤔 If they are not set, the join requests won't work because the backend cannot manage the DU?

Reply:
Agents are handled correctly in client code already: https://github.com/streamr-dev/streamr-client-javascript/blob/21d668d228c560a0569908c19329220f0235b2a7/src/dataunion/DataUnion.ts#L507. We could handle it here also but owner address must be added too.

Reply:
Ok, better this way to let the client handle it 👍